.github/workflows/ci_test_package.yml

```diff
@@ -38,21 +38,23 @@ jobs:
       fail-fast: false # Don't fail one DWH if the others fail
       matrix:
         warehouse: ["snowflake", "bigquery"]
-    runs-on: ubuntu-latest
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     environment:
       name: Approve Integration Tests
     permissions:
       contents: "read"
       id-token: "write"
 
     steps:
-      - name: Get latest release
-        uses: rez0n/actions-github-release@main
+      - name: Checkout
         id: latest_release
-        env:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          repository: ${{ github.repository }}
-          type: "stable"
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: |
+          output=$(git describe --tags --abbrev=0)
+          echo "::set-output name=release::$output"
```

Comment:
Remind me @jared-rimmer - what was the aim of this change? It's been a while since I looked at this.

Reply:
I have two slight concerns with it:
1. I think `set-output` is being deprecated in favour of `$GITHUB_OUTPUT` instead.
2. This won't allow for non-stable releases, or if we tag with something else. If this is the only way of getting it to work with Powershell, it might be worth having this only when it's PowerShell, and then retaining the original action for ubuntu

What do you think?

Reply:
Yep your solution sounds good!